src/swap/saga.test.ts

```diff
@@ -61,26 +59,13 @@ describe(swapSubmitSaga, () => {
     jest.clearAllMocks()
   })
 
-  const mockResponse = {
-    ok: true,
-    json: () => {
-      return { validatedSwapTransaction: mockSwapTransaction }
-    },
-  }
-
-  const buyToken = '0xd8763cba276a3738e6de85b4b3bf5fded6d6ca73'
-  const sellToken = '0xe8537a3d056da446677b9e9d6c5db704eaab4787'
-  const buyAmount = '10000000000000000'
-  const executeSwapUri = `${networkConfig.executeSwapUrl}?buyToken=${buyToken}&sellToken=${sellToken}&buyAmount=${buyAmount}&userAddress=${mockAccount}`
```

Comment:
i think `executeSwapUrl` can also be removed from networkConfig

Reply:
Fixed in 8f39cc75470d2b6d2ccf4f8d83e66d187f639b4b & 2831d9770994f63a05b1a2bf66339f9818c09f7d!